src/main/java/domain/attendance/AttendanceDate.java

```diff
@@ -0,0 +1,69 @@
+package domain.attendance;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import util.DateTimeUtil;
+
+public class AttendanceDate implements Comparable<AttendanceDate> {
+    public static final int SATURDAY = 6;
+
+    private static final int DEFAULT_START_TIME = 2024;
+    private static final int DEFAULT_START_MONTH = 12;
+    private static final int DEFAULT_START_DAY = 2;
+
+    public static final LocalDate DEFAULT_START_DATE = java.time.LocalDate.of(DEFAULT_START_TIME, DEFAULT_START_MONTH,
```

Comment:
기본 시작일을 상수로 생성해놓고 재사용 👍 
패키지 경로는 제거해도 되겠군요!

```suggestion
    public static final LocalDate DEFAULT_START_DATE = LocalDate.of(DEFAULT_START_TIME, DEFAULT_START_MONTH,
```

Reply:
엇 놓친 부분입니다 발견해주셔서 감사해요!!